Packs/TaniumThreatResponse/Integrations/TaniumThreatResponseV2/TaniumThreatResponseV2.py

```diff
@@ -789,6 +799,40 @@ def update_intel_doc(client: Client, data_args: dict) -> Tuple[str, dict, Union[
     return human_readable, outputs, raw_response
 
 
+def delete_intel_doc(client, data_args):
+    params = {
+        'id': data_args.get('intel_doc_id')
+    }
+    raw_response = client.do_request('DELETE', '/plugin/products/detect3/api/v1/intels/', params=params)
+
+    return 'Intel Doc deleted', {}, raw_response
```

Comment:
One example of when a delete would fail is if the intel doc with specified ID does not exists; if this occurs, `return_error` will be called within the `main` function on line 2108 which returns the error `Error in Tanium Threat Response Integration: b'{"error":"not found"}'`